zmsapi/tests/Zmsapi/WarehouseSubjectListGetTest.php

```
@@ -11,7 +11,7 @@ class WarehouseSubjectListGetTest extends Base
     public function testRendering()
     {
         $workstation = $this->setWorkstation();
-        $workstation->getUseraccount()->setRights('scope');
+        $workstation->getUseraccount()->setPermissions('statistic') = true;
```

PHPStan (2.1.51)

[error] Expression on left side of assignment is not assignable.


(assign.invalidExpr)